tests/Database/DatabaseEloquentModelTest.php

```diff
@@ -114,26 +114,12 @@ public function testForceCreateMethodSavesNewModelWithGuardedAttributes()
 	}
 
 
-	public function testFindMethodCallsQueryBuilderCorrectly()
```

Comment:
Can you not remove the tests please. Just because it's deferring to somewhere else, it doesn't mean we shouldn't have tests to check that this context is working.